### DISCUSSION TITLE
Features without geometry

### DISCUSSION THREAD

**Discussion by zvfcs** [ldproxy/ldproxy#738] [Configuration] [answered]
Hi,
I'm working on a OGC API for cultural heritage sites in Norway. Most of the features in our database have geometries, but there are still quite a lot of features that don't. Maybe they will get that information in the future, or maybe not. What I'm wondering is if those features can be part of a OGC API. As I understand the geoJSON standard, a geometry field with content is mandatory. If I add a source table to ldproxy with rows, with and without geometries, the features without geometry is included in the API, but with "geometry: null". Is that actually allowed?

It's too bad if an entirely different API is needed for those geometry-less features, so I guess I'm just hoping there's a solution for delivery of "mixed" data.

**Comment by cportele** [comment, marked as answer]
Hi Erik,

"geometry" can be "null", this is valid GeoJSON. 

See https://www.rfc-editor.org/rfc/rfc7946#section-3.2:

> A Feature object has a member with the name "geometry".  The value of the geometry member SHALL be either a Geometry object as defined above or, in the case that the Feature is unlocated, a JSON null value. 

Best regards, Clemens

**Comment by zvfcs** [comment]
Thank you for your reply! Seems like I misread some other part of the specification.
